tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -98,28 +99,74 @@ Verify Documentation Link HTTP Status Code
     ...    also checks the RHODS dcoumentation link present in resource page.
     [Tags]    Sanity
     ...       ODS-327    ODS-492
-    ${links}=  Get RHODS Documentation Links From Dashboard
-    Check External Links Status     links=${links}
+    ${links}=    Get RHODS Documentation Links From Dashboard
+    Check External Links Status    links=${links}
 
 Verify Logged In Users Are Displayed In The Dashboard
     [Documentation]    It verifies that logged in users username is displayed on RHODS Dashboard.
     [Tags]    Sanity
-    ...       ODS-354
-    ...       Tier1
-    Verify Username Displayed On RHODS Dashboard   ${TEST_USER.USERNAME}
+    ...       ODS-354    Tier1
+    Verify Username Displayed On RHODS Dashboard    ${TEST_USER.USERNAME}
 
 Search and Verify GPU Items Appears In Resources Page
-    [Tags]  ODS-1226
-    Search Items In Resources Section   GPU
+    [Tags]    ODS-1226
+    Search Items In Resources Section    GPU
     Check GPU Resources
 
+Verify filters Are Working On Resources Page
+    [Documentation]    check if it is possible to filter items by enabling various filters like status,provider
+    [Tags]    Sanity
+    ...       ODS-489
+    ...       Tier1
+    Click Link    Resources
+    Wait Until Page Contains    Self-managed
+    #Filter by enabled filter
+    Select Checkbox Using Id    enabled-filter-checkbox--check-box
+    @{expected_items}=    Create List    Creating a Jupyter notebook
+    ...    Deploying a sample Python application using Flask and OpenShift
+    ...    How to install Python packages on your notebook server    How to update notebook server settings
+    ...    How to use data from Amazon S3 buckets    How to view installed packages on your notebook server
+    ...    JupyterHub
+    Verify The Cards Using Selector And It Value    selector=pf-c-card__title odh-card__doc-title
+    ...    list_of_items=${expected_items}
+    Deselct Checkbox Using Id    enabled-filter-checkbox--check-box
+    # Filter by application (aka provider)
+    @{expected_items}=    Create List    by Anaconda Commercial Edition
+    Select Checkbox Using Id    Anaconda Commercial Edition--check-box
+    Verify The Cards Using Selector And It Value    selector=pf-c-card__title odh-card__doc-title
+    ...    list_of_items=${expected_items}    index_of_text=1
```

Comment:
maybe you can define this list (and all the others) in the variables section of RF

Reply:
Is ok if I do not change the declaration of lists
maybe if we declare in the variable section there will be too many variable with different names

Reply:
I think it wouldn't be a big problem to have 5 variables in the variable sections